src/NetCDF.jl

```diff
@@ -259,7 +265,7 @@ function nc_get_vara_x!{T<:Union{UTF8String,ASCIIString}}(ncid::Integer,varid::I
   retvalsa
 end
 
-function nc_get_var1_x(ncid::Integer,varid::Integer,start::Vector{UInt},::Union{Type{ASCIIString},Type{UTF8String}})
+function nc_get_var1_x(ncid::Integer,varid::Integer,start::Vector{UInt},::Type{AbstractString})
```

Comment:
Same here, this would not work for String types since isa(Type{ASCIIString},Type{AbstractString}) returns false...